ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -121,10 +132,36 @@ export const CreateGkeApp = ({
     autopauseThreshold: undefined,
   };
 
-  const createAppRequest: CreateAppRequest = {
-    ...defaultConfig,
-    persistentDiskRequest,
-  };
+  const [createAppRequest, setCreateAppRequest] =
+    React.useState<CreateAppRequest>({
+      ...defaultConfig,
+      persistentDiskRequest,
+    });
+
+  const [autodeleteDropdownChecked, setAutodeleteDropdownChecked] =
+    React.useState(false);
+
+  const autodeleteRemainingDays: number = (() => {
+    if (app?.autodeleteEnabled && app.dateAccessed && app.autodeleteThreshold) {
```

Comment:
I started RStudio but did not yet access/open the app. When I went to open the RStudio panel, the label showed just "0". Do we need to have dateAccessed here?
![image](https://github.com/all-of-us/workbench/assets/24514630/9cfeb68a-98a4-4620-bbca-37b33b0c1dd8)

Reply:
Interesting. The app will be autodeleted after 7 days even if you never open it? I can not recreate this issue on my side. When I have an app not accessed, I see the correct label. Do you know where does that`0` come from?